packages/cli/generators/app/templates/src/application.ts.ejs

```diff
@@ -1,5 +1,9 @@
 import {BootMixin} from '@loopback/boot';
 import {ApplicationConfig} from '@loopback/core';
+import {
+  RestExplorerBindings,
+  RestExplorerComponent,
+} from '@loopback/rest-explorer';
```

Comment:
should we add `this.component(RestExplorerComponent)` in the generated application class as well?

Reply:
This is resolved in a later commit.